modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm

```diff
@@ -0,0 +1,8 @@
+//////////////////////////////////////
```

Comment:
the box is unnecessary, volra is big dumb, just do a normal comment on the same line as the datum

Reply:
i kinda like the box tho its quirky